clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3644,13 +3645,28 @@ static void RenderSSPOptions(const Driver &D, const ToolChain &TC,
           << A->getOption().getName() << Value << "sysreg global";
       return;
     }
+    if (EffectiveTriple.isRISCV()) {
+      if (Value != "tls" && Value != "global") {
```

Comment:
Is this also enabling "global" as a value? Is that supported in the backend?

Reply:
Yes, that's the existing non-TLS version which uses `__stack_chk_guard`. It's the default mode. I could add a test to make sure it still works?

Reply:
Added a test to make sure -mstack-protector-guard=global still works correctly after the patch. Also added positive tests to make sure valid driver arguments get passed through to the backend.